docs/getting_started.rst

```diff
@@ -0,0 +1,43 @@
+=================================================
+Getting Started
+=================================================
+
+This SDK is pre-installed on pi-topOS, so you don't need to install it manually.
```

Comment:
All of this section should go under an "Installing the SDK" header

Reply:
More emphasis should be placed on "plug and play on pi-topOS" - we should link to the OS download page, and encourage people to do that.

Reply:
We should probably put something in here as a "checking that everything is working" step - e.g. check that the device manager is running, and producing a Hello World

Reply:
Then we should direct people to the examples